StarterProject/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node":true
+    },
+    "globals": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "double"
+        ],
+        "semi": [
+            "warn",
+            "always"
+        ]
+    }
+}
```

Comment:
Since this is a sample project, I'm not sure we need eslint at all -- everyone has their own preferences anyway 😛

Reply:
I added it in here as not all of us know how to configure linters, and on some automation projects, the automation folders are separate from the parts with code.